resources/src/commonMain/kotlin/dev/icerock/moko/resources/ColorResource.kt

```diff
@@ -6,8 +6,8 @@ package dev.icerock.moko.resources
 
 import dev.icerock.moko.graphics.Color
 
-sealed class ColorResource {
-    class Single(val color: Color) : ColorResource()
+sealed class ColorResource(val name: String) {
```

Comment:
name have no sense for not apple targets. we should change `ColorResource` to expect/actual. for apple target we should have `name` and `bundle` args.
for android we should use resource arg, with generation of xml colors file.
for other targets - save current way with manual selection

Reply:
fixed in #439